vnext/Microsoft.ReactNative/Views/ShadowNodeBase.h

```diff
@@ -121,10 +121,17 @@ struct REACTWINDOWS_EXPORT ShadowNodeBase : public ShadowNode {
     return m_onMouseEnterRegistered || m_onMouseLeaveRegistered;
   }
 
+  void IsAccessible(bool accessible);
+  bool IsAccessible();
+  void IsFocusable(bool focusable);
+  bool IsFocusable();
+
  protected:
   XamlView m_view;
   bool m_updating = false;
   comp::CompositionPropertySet m_transformPS{nullptr};
+  bool m_isFocusable = true;
```

Comment:
Possibly not a huge concern, but does it really make sense to pay for these two booleans for any shadow node that is not focusable? E.g., Text is another frequently used component that now is 8 bytes larger because of this change.

In reality, if most apps are 10-20% text nodes (even that seems high), and there are, say 10,000 nodes in a very complex app (also a high estimate), you're talking about 2-4KB of additional memory, so it's probably nothing to worry about.

Reply:
We might be able to micro-optimize this by putting the bools up next to `m_updating`. Would have to look at the layout of the class after compilation.